### PR TITLE
Redmine #8592: Clarify DHCP hostname registration

### PR DESCRIPTION
**Pull request by imcdona**
- [x] Redmine Issue: https://redmine.pfsense.org/issues/8592
- [x] Ready for review: yes

### CODE REVIEW

File: src/usr/local/www/services_dnsmasq.php [hunk 1]
```diff
@@ -286,8 +286,8 @@ function build_if_list() {
 	'Static DHCP',
 	'Register DHCP static mappings in DNS forwarder',
 	$pconfig['regdhcpstatic']
-))->setHelp('If this option is set, DHCP static mappings will '.
-					'be registered in the DNS forwarder, so that their name can be '.
+))->setHelp('If this option is set, IPv4 DHCP static mappings, will '.
```
Comment (jim-p) [inline src/usr/local/www/services_dnsmasq.php]: No need for a comma after "mappings"

File: src/usr/local/www/services_unbound.php [hunk 2]
```diff
@@ -412,8 +412,8 @@ function build_if_list($selectedifs) {
 	'DHCP Registration',
 	'Register DHCP leases in the DNS Resolver',
 	$pconfig['regdhcp']
-))->setHelp('If this option is set, then machines that specify their hostname when requesting a DHCP lease will be registered'.
-					' in the DNS Resolver, so that their name can be resolved.'.
+))->setHelp('If this option is set, then machines that specify their hostname when requesting an IPv4 DHCP lease, will be registered'.
```
Comment (jim-p) [inline src/usr/local/www/services_unbound.php]: No need for a comma after "lease"

**Comment by imcdona** [comment]
Do I need to make changes and re-submit or can you accept the changes that you've already made?

**Comment by jim-p** [comment]
If you make a commit to your branch and push it should keep it on this same PR.

**Pull request merged**